index.html

```diff
@@ -135,13 +173,17 @@ <h2 class="professional">Professional Art Printing Data</h2>
             
           </div>
         </section>
-      </section>
+    </section>
 
-       <section class="about">
+  <section class="about">
+    <section class="about-headline">
     <h1 class="about-title">About me</h1>
-    <p class="about-me">Hello I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="about-me">Hey there! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
     <button class="resume-link"><a href="#">Get My Resume</a></button>
-  
+    </section>
+
+      <div class="divider"></div>
+
     <section class="lists">
 
```

Comment:
This is from FIGMA design:
![image](https://user-images.githubusercontent.com/61361037/213742178-2e182dd3-cae5-44d3-a35c-96422497cfca.png)


This is your current bullet points:
![image](https://user-images.githubusercontent.com/61361037/213741978-dc0aa79f-f0c1-4766-8ca2-8bc007b7fafa.png)

You just have tp export these icons, and apply them there.